data/trie/doubleListSync.go

```diff
@@ -30,7 +30,7 @@ type doubleListTrieSyncer struct {
 	handlerID                 string
 	trieSyncStatistics        data.SyncStatisticsHandler
 	lastSyncedTrieNode        time.Time
-	timeoutBetweenCommits     time.Duration
+	timeoutNodeReceived       time.Duration
```

Comment:
`receivedNodesTimeout` ? in the entire PR

Reply:
done